src/routes/safe/components/Settings/ManageOwners/RemoveOwnerModal/screens/Review/index.tsx

```diff
@@ -56,6 +56,8 @@ export const ReviewRemoveOwnerModal = ({
   const owners = useSelector(safeOwnersSelector)
   const addressBook = useSelector(addressBookSelector)
   const ownersWithAddressBookName = owners ? getOwnersWithNameFromAddressBook(addressBook, owners) : List([])
+  const [manualSafeTxGas, setManualSafeTxGas] = useState(0)
+  const [manualGasPrice, setManualGasPrice] = useState<string | undefined>()
```

Comment:
I think we are repeating this code many times, perhaps if we add a parameter in useTransactionParameters indicating if the parameters were modified, it could simplify things.